tools/gcms2isocor/gcms2isocor.xml

```diff
@@ -0,0 +1,42 @@
+<tool id="gcms2isocor" name="Conversion GCMS PostRun Analysis to IsoCor" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="8.0.121">openjdk</requirement> 
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       ins="" 
+       #for $input in $input_gcms_files
+         #if $input
+           ins=\$ins" "'$input';
```

Comment:
I'd replace this line by `#set $ins = $ins + " " + $input`

Reply:
thes new line give return an error with planemo t
```xml
<command detect_errors="exit_code"><![CDATA[
       #set $ins = ""
       #for $input in $input_gcms_files
         #if $input
           #set $ins = $ins + " " + $input
         #end if
        #end for

        java -jar '$__tool_directory__/P2M2WorkflowsTools-assembly-0.1.jar' $ins --out '$output_isocor'
    ]]></command>
```
```bash
Traceback (most recent call last):
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/jobs/runners/__init__.py", line 237, in prepare_job
    job_wrapper.prepare()
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/jobs/__init__.py", line 1137, in prepare
    self.command_line, self.extra_filenames, self.environment_variables = tool_evaluator.build()
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/tools/evaluation.py", line 463, in build
    raise e
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/tools/evaluation.py", line 459, in build
    self.__build_command_line()
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/tools/evaluation.py", line 484, in __build_command_line
    command_line = fill_template(command, context=param_dict, python_template_version=self.tool.python_template_version)
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/util/template.py", line 107, in fill_template
    raise first_exception or e
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/util/template.py", line 80, in fill_template
    return unicodify(t, log_exception=False)
  File "/media/olivier/hdd-local/workspace/INRAE/galaxy/lib/galaxy/util/__init__.py", line 1059, in unicodify
    value = str(value)
  File "/home/olivier/anaconda3/lib/python3.8/site-packages/Cheetah/Template.py", line 1053, in __unicode__
    return getattr(self, mainMethName)()
  File "cheetah_DynamicallyCompiledCheetahTemplate_1632463899_506464_90204.py", line 90, in respond
NameMapper.NotFound: cannot find 'ins'
FAIL

```

Reply:
hu, strange... can you commit this so I can get more logs?